mentoring/templates/html/dashboard_report.html

```diff
@@ -0,0 +1,17 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
I don't really know what the right thing to do is. Honestly if you're not delivering translations, this block will always be in English so technically this is correct.